packages/vhd-lib/Vhd/VhdSynthetic.js

```diff
@@ -43,6 +43,16 @@ const VhdSynthetic = class VhdSynthetic extends VhdAbstract {
     }
   }
 
+  get compressionType() {
+    const compressionType = this.vhds[0].compressionType
+    for (let i = 0; i < this.vhds.length; i++) {
+      if (compressionType !== this.vhds[i].compressionType) {
+        return undefined
```

Comment:
Doesn't `undefined` mean no compression?

I believe there are 3 acceptable results to this situation:
- throw an error
- return the compression of the last VHD in the chain
- return the compression of the first VHD in the chain

What is the best answer in our situation?